src/Bicep.Core/Parsing/BaseParser.cs

```diff
@@ -38,47 +38,33 @@ protected BaseParser(string text)
 
         private static bool CheckKeyword(Token? token, string keyword) => token?.Type == TokenType.Identifier && token.Text == keyword;
 
-        private static int GetOperatorPrecedence(TokenType tokenType)
+        private static int GetOperatorPrecedence(TokenType tokenType) => tokenType switch
         {
             // the absolute values are not important here
-            switch (tokenType)
-            {
-                case TokenType.Modulo:
-                case TokenType.Asterisk:
-                case TokenType.Slash:
-                    return 100;
-
-                case TokenType.Plus:
-                case TokenType.Minus:
-                    return 90;
-
-                case TokenType.GreaterThan:
-                case TokenType.GreaterThanOrEqual:
-                case TokenType.LessThan:
-                case TokenType.LessThanOrEqual:
-                    return 80;
+            TokenType.Modulo or
+            TokenType.Asterisk or
+            TokenType.Slash => 100,
 
-                case TokenType.Equals:
-                case TokenType.NotEquals:
-                case TokenType.EqualsInsensitive:
-                case TokenType.NotEqualsInsensitive:
-                    return 70;
+            TokenType.Plus or
+            TokenType.Minus => 90,
 
-                // if we add bitwise operators in the future, they should go here
+            TokenType.GreaterThan
+            or TokenType.GreaterThanOrEqual
```

Comment:
>or [](http://example.com/codeflow?start=12&length=2)

nit - The `or` formatting is inconsistent between L44-46 and the others.

Reply:
Good catch. Fixed.